File: beginner/javascript/1043.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split(' ');

var a = parseFloat(valores.shift());
var b = parseFloat(valores.shift());
var c = parseFloat(valores.shift());

var perimetro = a + b + c;
var area = (a + b) * c / 2;

if((a < (b + c)) && (b < (a + c)) && (c < (a + b))){
    console.log("Perimetro = " + perimetro.toFixed(1));
} else{
    console.log("Area = " + area.toFixed(1));
}

File: beginner/c/1043.c
#include <stdio.h>

int main(){

    double a, b, c, perimetro, area;

    scanf("%lf %lf %lf", &a, &b, &c);

    perimetro = a + b + c;
    area = (a + b) * c / 2;

    if((a < (b + c)) && (b < (a + c)) && (c < (a + b))){
        printf("Perimetro = %.1lf\n", perimetro);
    } else{
        printf("Area = %.1lf\n", area);
    }

    return 0;
}
